### PR TITLE
add color field to group model

### PR DESCRIPTION
**Pull request by atreidas1**

### CODE REVIEW

File: items/migrations/0011_merge.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('items', '0010_auto_20150825_0842'),
+        ('items', '0010_file'),
+    ]
+
+    operations = [
+    ]
```
Comment (NikitaSedyx) [inline items/migrations/0011_merge.py:15]: for what it migration?

File: items/migrations/0013_auto_20150826_0725.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+    dependencies = [
+        ('items', '0012_group_color'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='group',
+            name='color',
+            field=models.CharField(default=b'FFFFFF', max_length=6, choices=[(b'FF0000', b'red'), (b'33CC00', b'green'), (b'00CCFF', b'blue'), (b'FF9900', b'orange'), (b'FFFFFF', b'white'), (b'FFFF00', b'yellow')]),
```
Comment (NikitaSedyx) [inline items/migrations/0013_auto_20150826_0725.py]: it's better to use class_name as value instead of hex code

Comment (NikitaSedyx) [inline items/migrations/0013_auto_20150826_0725.py]: and break line, pls

File: items/models.py [hunk 3]
```diff
@@ -12,6 +12,21 @@ class Group(models.Model):
   title = models.CharField(max_length=150, default='Group Tasks')
   view = models.BooleanField(default=True)
   is_deleted = models.BooleanField(default=False)
+  RED = 'FF0000'
+  GREEN = '33CC00'
+  BLUE = '00CCFF'
+  ORANGE = 'FF9900'
+  WHITE = 'FFFFFF'
+  YELLOW = 'FFFF00'
```
Comment (ghost) [inline items/models.py]: IMHO, it's not the coolest colours. Could you please try to use some colour generators? http://paletton.com or http://www.colorsontheweb.com/colorwizard.asp or anything else. 
Also it should be easy-expandable solution

File: items/migrations/0011_merge.py (hunk 1 above)
Comment (atreidas1) [inline items/migrations/0011_merge.py:15]: it's migration offer django after merging with master

**Comment by ghost** [comment]
What's with @NikitaSedyx 's comment https://github.com/NikitaSedyx/todo-dj/pull/28#discussion-diff-37954718?

File: items/models.py [hunk 4]
```diff
@@ -12,6 +12,21 @@ class Group(models.Model):
...
   view = models.BooleanField(default=True)
   is_deleted = models.BooleanField(default=False)
+  RED = 'red'
+  GREEN = 'green'
+  BLUE = 'blue'
+  ORANGE = 'orange'
+  WHITE = 'white'
+  YELLOW = 'yellow'
+  COLORS = (
+    (RED, 'red'),
```
Comment (ghost) [inline items/models.py]: it's confusing. If you have separate variables, you can use them. Like `(RED, RED),` and so on

File: items/migrations/0013_auto_20150826_0725.py (hunk 2 above)
Comment (ghost) [inline items/migrations/0013_auto_20150826_0725.py]: maybe you have to revert this migration ?

**Pull request merged**